cmd/vaultRotateSecretId.go

```diff
@@ -89,26 +90,41 @@ func runVaultRotateSecretID(utils vaultRotateSecretIDUtils) error {
 		log.Entry().Infof("Your secret ID is about to expire in %.0f days", ttl.Round(time.Hour*24).Hours()/24)
 	}
 
+	// Check if the secret store is ADO and apply the TTL condition
+	if config.SecretStore == "ado" {
+		warnMessage := "ADO Personal Access Token is required but not provided. Secret ID rotation cannot proceed for Azure DevOps.\n" +
+			"Note: In Azure DevOps, Vault secrets are rotated automatically by the 'automaticd' service when the TTL is 18 days or less."
```

Comment:
Suggestion :
If pipeline is connected to 'automaticd' , by default Vault secret rotation is handled automatically by 'automaticd' . In other cases Piper has to rotate the secrets , a valid PAT is then necessary to continue.